### PR TITLE
contrast: remove default SVNs for TDX

### PR DESCRIPTION
**Pull request by Freax13**
We don't ship minimum default SVNs for bare metal, so we shouldn't have a default for minimumTeeTcbSvn. Having a default for mrSeam is problematic because it legitimately can change as a customer updates their firmware.

### CODE REVIEW

**Comment by katexochen** [comment]
I think we need documentation to instruct users how to fill these values if we don't have defaults, right?

File: e2e/internal/contrasttest/contrasttest.go [hunk 1]
```diff
@@ -208,6 +208,14 @@ func (ct *ContrastTest) patchReferenceValues(t *testing.T, platform platforms.Pl
 			snp.MinimumTCB.MicrocodeVersion = toPtr(manifest.SVN(0))
 			m.ReferenceValues.SNP[i] = snp
 		}
+	case platforms.K3sQEMUTDX, platforms.RKE2QEMUTDX:
+		// The generate command doesn't fill in all required fields when
+		// generating a manifest for baremetal TDX. Do that now.
```
[the remark this answers is not in the document]
Comment (Freax13) [reply e2e/internal/contrasttest/contrasttest.go:213]: This code isn't executed when running the default just target and the code in the justfile isn't executed when running the e2e tests. In both cases, this only affects our development workflows and so we don't expect this to change unless we upgrade the firmware on our server.

File: docs/docs/deployment.md [hunk 2]
```diff
@@ -211,11 +211,21 @@ contrast generate --reference-values aks-clh-snp resources/
 ```sh
 contrast generate --reference-values k3s-qemu-snp resources/
 ```
+:::note[Missing TCB values]
+On bare metal SEV-SNP, `contrast generate` is unable to fill in the `MinimumTCB` values as they can vary between platforms.
+They will have to be filled in manually.
+If you don't know the correct values use `{"BootloaderVersion":255,"TEEVersion":255,"SNPVersion":255,"MicrocodeVersion":255}` and observe the correct values in the error messages in the following steps. Note that the values will differ between CPU models.
```
[the remark this answers is not in the document]
Comment (Freax13) [reply docs/docs/deployment.md:217]: > Nit: I feel the word "correct" needs some explanation around trust-on-first-use. 

Will do.

> Even better would be some instructions for deriving the values directly on the host.

AFAICT there's no interface to ask the host kernel about the active values and I don't think the TDX module even exposes that. AFAICT the only way to get the values is to launch a virtual machine and create a quote. While this is technically feasible, as of right now, I don't think there's existing tooling for users to do this, though I'd be more than happy to write some.

[the remark this answers is not in the document]
Comment (burgerdev) [reply docs/docs/deployment.md]: It would sure be great to have something like `snphost` for TDX.

**Review by burgerdev — APPROVED** [review]

**Pull request merged**